apps/meteor/client/views/admin/moderation/helpers/DateRangePicker.tsx

```diff
@@ -76,9 +70,13 @@ const DateRangePicker = ({ onChange }: DateRangePickerProps) => {
 		}
 	});
 
+	useEffect(() => {
+		handleOptionClick(defaultSelectedKey);
```

Comment:
A tree of function calls and a state mutation just for a default value?

<img src="https://media1.giphy.com/media/v1.Y2lkPWJkM2VhNTdlNm11cnlrdWtvMms3b25qam1tYzNxNnJ6OWRrdGUwZ2hneW44Y25ydiZlcD12MV9naWZzX3NlYXJjaCZjdD1n/18gEqArCQNlo5zowZn/giphy.gif"/>